robot/Cumulus/resources/GAUPageObject.py

```diff
@@ -1,8 +1,31 @@
 from cumulusci.robotframework.pageobjects import ListingPage
 from cumulusci.robotframework.pageobjects import pageobject
+from cumulusci.robotframework.pageobjects import BasePage
 from BaseObjects import BaseNPSPPage
 from NPSP import npsp_lex_locators
 
+
+@pageobject("Custom", "GauAllocation")
```

Comment:
how about "ManageAllocations" instead as its more relevant? I have same pageobject in my PR but I named like this to be more specific. If you are ok with my suggestion, I can merge my PR after urs and resolve the conflict or u can update the name.